consensus/test_data/build.sh

```diff
@@ -1,14 +1,26 @@
 #!/usr/bin/env bash
 
 # XXX: removes tendermint dir
+# TODO: does not work on OSX
```

Comment:
i think I figured it out - you need to install `gnu-sed`. eg. `brew install gnu-sed --with-default-names` ... that should make your `sed` point to `usr/local/bin/sed` which is actually `gnu-sed`. 

We should document this better, but eg. https://www.topbug.net/blog/2013/04/14/install-and-use-gnu-command-line-tools-in-mac-os-x/

Reply:
Cool! Thanks

Reply:
Actually, the script does not contain sed anymore, so we can remove this line safely. But good to know anyway!